src/libraries/System.Private.CoreLib/src/System/Diagnostics/StackTrace.cs

```diff
@@ -18,6 +18,7 @@ public partial class StackTrace
     {
         public const int METHODS_TO_SKIP = 0;
 
+        private static readonly bool s_ilOffsetToStackTrace = Environment.GetEnvironmentVariable("COMPlus_ILOffsetToStackTrace") == "1";
```

Comment:
Lets make this on by default and use this env var to opt-out if there is ever a problem with the new formatting. 

It should be as simple as changing the condition to `!= "0"`.

Reply:
Basically, I wanted the stactrack output to remain unchanged unless the environment variable was set.
Do you think it's okay if the basic stacktrace output method is changed?

Reply:
I was thinking that having the IL offset/token display on by default if there are no pdbs won't affect anything else, but there could be code that scraps the stack trace text not expecting il offset/token. Lets leave this check the way you have it.

Reply:
I agree that it does not sound like a good idea to show these cryptic numbers by default. 99% of users won't know what to do with them and will just get confused.